x-pack/plugins/security_solution/public/resolver/store/selectors.ts

```diff
@@ -240,12 +241,13 @@ export const originID: (state: ResolverState) => string | undefined = composeSel
  * Takes a nodeID (aka entity_id) and returns the node ID of the node that aria should 'flowto' or null
  * If the node has a flowto candidate that is currently visible, that will be returned, otherwise null.
  */
-export const ariaFlowtoNodeID: (
-  state: ResolverState
-) => (time: number) => (nodeID: string) => string | null = createSelector(
+export const ariaFlowtoNodeID = createSelector(
   visibleNodesAndEdgeLines,
   composeSelectors(dataStateSelector, dataSelectors.ariaFlowtoCandidate),
-  (visibleNodesAndEdgeLinesAtTime, ariaFlowtoCandidate) => {
+  function (
```

Comment:
Couldn't get this to work with an arrow function, TypeScript stopped complaining as soon as I turned this into an explicit `function` declaration instead.